lambdas/postgres-migration-async-operation/src/index.ts

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const asyncOperations = require('@cumulus/async-operations');
+const { models } = require('@cumulus/api');
+const Logger = require('@cumulus/logger');
+
+const logger = new Logger({ sender: '@cumulus/postgres-migration-async-operation' });
+exports.handler = async () => {
+  const stackName = process.env.stackName;
+  const systemBucket = process.env.system_bucket;
+  const tableName = process.env.AsyncOperationsTable;
+
+  logger.info('Starting Postgres Migration Async Operation');
+  const asyncOperation = await asyncOperations.startAsyncOperation({
+    cluster: process.env.EcsCluster,
+    lambdaName: process.env.MigrationLambda,
+    asyncOperationTaskDefinition: process.env.AsyncOperationTaskDefinition,
+    description: 'Data Migration 2 Lambda ECS Run',
+    operationType: 'Data Migration 2',
+    payload: {
+    },
+    stackName,
+    systemBucket,
+    dynamoTableName: tableName,
```

Comment:
Pair of the suggested change on L11
```suggestion
    dynamoTableName,
```

Reply:
[`e0b53d0` (#2165)](https://github.com/nasa/cumulus/pull/2165/commits/e0b53d0caad6c024a490901ccc2670b669500cb5)